app/src/main/java/ch/epfl/polychef/recipe/Recipe.java

```diff
@@ -206,6 +209,23 @@ public String getRecipeUuid(){
         return recipeUuid;
     }
 
+    /**
+     * Returns the key used by the database to store the recipe.
+     * @return the key used by the database to store the recipe
+     */
+    public String getRecipeDatabaseKey(){
```

Comment:
Add `@Exclude` tag to the method declaration (_getRecipeDatabaseKey_ and _setRecipeDatabaseKey_), otherwise it will also be put on Firebase and the key will appears twice in the database

Reply:
It's done now if I correctly understood